src/test/scala/ca/ferlab/ferload/client/commands/ConfigureTest.scala

```diff
@@ -0,0 +1,77 @@
+package ca.ferlab.ferload.client.commands
+
+import ca.ferlab.ferload.client.clients.inf.{ICommandLine, IFerload}
+import ca.ferlab.ferload.client.configurations._
+import com.typesafe.config.{Config, ConfigFactory}
+import org.json.JSONObject
+import org.scalatest.BeforeAndAfter
+import org.scalatest.funsuite.AnyFunSuite
+import picocli.CommandLine
+
+import java.io.File
+import java.util
+
+class ConfigureTest extends AnyFunSuite with BeforeAndAfter {
+
+  val appTestConfig: Config = ConfigFactory.load.getObject("ferload-client").toConfig
+  val path: String = File.createTempFile(".ferload-client", ".properties").getAbsolutePath
+  val mockUserConfig = new UserConfig(path)
+  val mockCommandLineInf: ICommandLine = new ICommandLine {
```

Comment:
If you need a mock command line for many tests, I would move the creation of this mock to a test util class with some param so you can re-use it for different cases.